weblayer/browser/java/org/chromium/weblayer_private/TabImpl.java

```diff
@@ -1313,5 +1340,8 @@ String registerWebMessageCallback(long nativeTabImpl, String jsObjectName,
         boolean isDesktopUserAgentEnabled(long nativeTabImpl);
         void download(long nativeTabImpl, long nativeContextMenuParams);
         void destroyContextMenuParams(long contextMenuParams);
+        String[] getContentFilterHosts(long nativeTabImpl);
```

Comment:
@yozuysal - maybe there is a smarter way to pass Map<String, Int> over JNI

Reply:
Looks like https://source.chromium.org/chromium/chromium/src/+/main:base/android/jni_array.h doesn't have anything useful. Folks on webrtc land seems to have gone rogue and added some [utilities](https://source.chromium.org/chromium/chromium/src/+/main:third_party/webrtc/sdk/android/native_api/jni/java_types.cc) that might be useful , but maybe it is an overkill wrt what you already have

Reply:
OK, thanks for looking!